src/session_concurrency/payload_dataset_manager.h

```diff
@@ -56,8 +58,6 @@ class PayloadDatasetManager {
 
   PayloadsMapType CreateSessionIdToPayloadsMap() const;
 
-  std::string GetSessionID(size_t dataset_index) const;
-
```

Comment:
`GetSessionID()` is moved from private to public since it is now called in the place outside this class where the session ID input is recorded for the profile export JSON.